src/import/planner.c

```diff
@@ -21,24 +21,28 @@
 #include <executor/nodeAgg.h>
 #include <nodes/makefuncs.h>
 #include <nodes/nodeFuncs.h>
-#include <optimizer/clauses.h>
 #include <optimizer/cost.h>
 #include <optimizer/paths.h>
 #include <optimizer/placeholder.h>
 #include <optimizer/planner.h>
-#include <optimizer/subselect.h>
 #include <optimizer/tlist.h>
-#include <optimizer/var.h>
 #include <parser/parsetree.h>
 #include <utils/datum.h>
 #include <utils/lsyscache.h>
 #include <utils/rel.h>
-#include <utils/selfuncs.h>
 
-#include "planner_import.h"
 #include "compat.h"
+#if PG12_LT
+#include <optimizer/clauses.h>
+#include <optimizer/subselect.h>
+#include <optimizer/var.h>
+#else
+#include <optimizer/optimizer.h>
+#endif
+
+#include "planner.h"
 
-#if (PG11 && PG_VERSION_NUM >= 110002) || (PG10 && PG_VERSION_NUM >= 100007) ||                    \
+#if PG12 || (PG11 && PG_VERSION_NUM >= 110002) || (PG10 && PG_VERSION_NUM >= 100007) ||            \
```

Comment:
@erimatnor should it be:
```suggestion
#if PG12_GE || (PG11 && PG_VERSION_NUM >= 110002) || (PG10 && PG_VERSION_NUM >= 100007) ||            \
```